.travis.yml

```diff
@@ -5,3 +5,15 @@ python:
 
 script:
 - pip install file://$(pwd)
+
+deploy:
+  provider: pypi
+  user: mottosso
+  distributions: "sdist bdist_wheel"
+  password:
+    secure: MOTTOSO-PUT-YOUR-PASSWORD-HERE
```

Comment:
Right here. :point_up:

Reply:
Haha, having a look now.. these passwords are harder to come by than you might expect! :)

Reply:
For my future self.

To get a password, you need Travis CLI.

**CentOS 7**

From e.g. https://labs.play-with-docker.com

```bash
$ yum update && yum install -y ruby ruby-devel
$ yum groupinstall -y "Development Tools"
$ gem install travis -v 1.8.9 --no-rdoc --no-ri
$ travis login
...
$ git clone https://github.com/pyblish/pyblish-nuke.git
$ cd pyblish-nuke
$ travis setup pypi
...
```

"Easy". :)

Reply:
"Easy", ha! Thanks for the instructions. It's something I was going to look into for future repos of my own and this is a helpful starting point.

Reply:
If you were to install the Travis CLI on your own computer, then you'd only need to do most of that only once. It's just that it doesn't work well on Windows (last I checked) and I'm always on different computers and OSes all the time. So the above is a "cloud based" version of doing it, which is what I've been doing the past few times. It's not often I need a password generated, so it's not too bad.

Just wish there was a web-ui for generating that instead.